pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -446,23 +451,19 @@ func registerClusterToCluster(r registry.Registry) {
 				m.Wait()
 
 				t.Status("waiting for replication stream to cutover")
+				retainedTime := getReplicationRetainedTime(t, setup.dst.sql, roachpb.TenantName(setup.dst.name))
```

Comment:
`SHOW TENANT WITH REPLICATION STATUS` stops working the minute the replication stream has stopped since the tenant no longer has an active replication job associated with it. You make a good point but I'm not too concerned in this particular test as the default retention is 25 hours, and our test runtime is nowhere close to that.